### ISSUE TITLE
no-duplicate-selectors bug

### ISSUE THREAD

**Issue by revyh** [AtomLinter/linter-stylelint#138]
Minimal setup to reproduce:

.stylelintrc.yml
```yaml
---
  rules:
    no-duplicate-selectors: [true, {severity: error}]
```

blah.css
```scss
.blah {
  color: red;
}
```

CLI output says that there is no error. I use `stylelint 5.1` and `linter-stylelint 2.0.1`.

Looks like text from editor is duplicated somehow when it's passed to stylelint. If that's true, it may explain #133.

Setup [blah.zip](https://github.com/AtomLinter/linter-stylelint/files/187444/blah.zip)

**Comment by Arcanemagus** [comment]
That is just _bizarre_, looking into this now.

**Comment by Arcanemagus** [comment]
@davidtheclark this looks like it might be a bug in `stylelint` itself, putting a breakpoint [here](https://github.com/AtomLinter/linter-stylelint/blob/v2.0.1/lib/index.js#L48) I'm seeing the correct options being passed into it:
![image](https://cloud.githubusercontent.com/assets/427137/14027675/18679c4a-f1b6-11e5-9f71-c3712a2b5e9b.png)

Seeing if I can isolate the issue further.

**Comment by Arcanemagus** [comment]
@davidtheclark This is definitely something funky going on with `stylelint`, putting a breakpoint [here](https://github.com/stylelint/stylelint/blob/5.1.0/src/standalone.js#L113) shows the CSS is _not_ being duplicated internally, but it is still triggering this rule.

Want an issue filed over there?

**Comment by davidtheclark** [comment]
Sure, thanks.

**Comment by Arcanemagus** [comment]
Filed it there.

**Comment by Arcanemagus** [comment]
v2.1.0 published with a version of `stylelint` including this fix.
